packages/frontend/.eslintrc.js

```diff
@@ -21,6 +21,12 @@ module.exports = {
     'ember/no-classic-classes': 0,
     'ember/no-mixins': 0,
     'ember/no-new-mixins': 0,
+    'n/no-extraneous-require': [
+      'error',
+      {
+        allowModules: ['broccoli-file-creator'],
```

Comment:
This feels dangerous, `embroider` tends to hate it when things are used without importing. I have no idea if it's possible, but can this dependency go in `packages/frontend/lib/new-version/package.json`?